admin/css/views/config.css

```
@@ -573,3 +656,343 @@ body.dark-theme .config-header {
+    display: flex;
+    flex-direction: column;
+    min-height: 0;
+}
+
+.pp-tweet-text {
+    margin: 0;
+    font-family: "JetBrains Mono", "Consolas", "Courier New", monospace;
+    font-size: 12.5px;
+    line-height: 1.6;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)